### ISSUE TITLE
Unable to create real GL context

### ISSUE THREAD

**Issue by Kraust** [benvanik/xenia#246]
I'm pretty new to this so bear with me.

- Tried to compile the current version of xenia, compiled okay.
- Tried to run the debug executable -> Unable to initialize GL context -> Unable to create real GL context

I'm unsure if this is just me not having some proper runtimes or what. I'm probably way out of line even throwing this issue up here, but I'm decently interested in the project.

Hardware: 
CPU: i5-4460
GPU: 270x
OS: Windows 10 Tech Preview x64 (I'm having a feeling it's related to this.)

Edit: Looking at the OpenGL spec, it looks like Xenia is set to require 4.5 instead of 4.4? Isn't this basically removing compatibility of AMD cards? AFAIK only Nvidia's cards support 4.5.

**Comment by Razzile** [comment]
I had this issue for a long time. then I reinstalled windows built xenia again and it was fine, no idea why.
(NVIDIA GTX 770)

**Comment by rlabrecque** [comment]
AMD started supporting OpenGL 4.5 with Catalyst 15.200.1018 (Windows 10 only)
http://www.geeks3d.com/forums/index.php/topic,3900.0.html

But Xenia is still broken on AMD cards currently at this point.

**Comment by Kraust** [comment]
I had initially closed this after I updated to 15.200 last night. I got the drivers off of Guru3D because Windows 10 didn't update for whatever reason

http://forums.guru3d.com/showthread.php?t=399743

I actually got xenia to work with moderate success. I do not have much to test, but it looks like there's *some* compatibility with AMD cards.

![image](https://cloud.githubusercontent.com/assets/6548135/8025284/f64cc2de-0d1d-11e5-85e6-d573a2bba32b.png)
![image](https://cloud.githubusercontent.com/assets/6548135/8025286/fc22053e-0d1d-11e5-818d-249cfbc4df3c.png)
